docs/azure.rst

```diff
@@ -8,43 +8,50 @@ Blob Storage
 ========
 Overview
 ========
-Azure Blob Storage is a cloud file storage system. It uses storage accounts to organize containers (similar to
-"buckets" for other storage providers) in which to store arbitrary files referred to as blobs.
 
-This connector currently only implements block blobs and not page or append blobs.
+`Azure Blob Storage <https://azure.microsoft.com/en-us/services/storage/blobs/>`_ is a cloud file storage system that
+uses storage accounts to organize containers (similar to "buckets" for other storage providers) in which to store
+arbitrary files referred to as 'blobs'. This Parsons integration currently only implements
+`block blobs, not page or append blobs <https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs>`_.
 
-You'll need credentials for an Azure Blob Storage storage account to use this connector. The ``azure-storage-blob``
-library is used for this connector, and `examples of how to create and use multiple types of credentials
-<https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-blob#types-of-credentials>`_
-are included in the documentation.
+.. note::
+  Authentication
+    This connector requires authentication credentials for an Azure Blob Storage storage account. The
+    ``azure-storage-blob`` library is used for this connector, and examples of how to create and use
+    `multiple types of credentials <https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-blob#types-of-credentials>`_
+    are included in the documentation.
 
 ==========
 Quickstart
 ==========
 
-**List containers and blobs**
+This class requires a ``credentials`` argument and *either* an ``account_name`` or an ``account_url`` argument that
+includes the account name. You can store these as environmental variables (``AZURE_CREDENTIAL``, ``AZURE_ACCOUNT_NAME``,
+and ``AZURE_ACCOUNT_URL``, respectively) or pass them in as arguments:
+
+**Instantiate class**
 
 .. code-block:: python
+   from parsons import AzureBlobStorage
 
-  from parsons import AzureBlobStorage
+   # First approach: Use API credentials via environmental variables
+   azure_blob = AzureBlobStorage()
 
-  azure_blob = AzureBlobStorage()
+   # Second approach: Pass API credentials as arguments
+   azure_blob = AzureBlobStorage(account_name='my_account_name', credential='1234')
```

Comment:
Is this missing the `account_url`?

Reply:
I don't think so, the class requires *either* `account_name` or `account_url`, since `account_url` is ultimately `f"https://{account_name}.blob.core.windows.net/"`